package.json

```diff
@@ -34,7 +34,7 @@
     "@sentry/browser": "^5.15.4",
     "@tippyjs/react": "^4.0.0-alpha.0",
     "@types/nprogress": "^0.2.0",
-    "@types/react-notify-toast": "^0.5.2",
+    "@types/react-notify-toast": "^0.5.0",
```

Comment:
@inyono did you downgrade this on purpose?

Reply:
Yes. My dependency workflow:
- Add the dependency with `^major.0.0` (resp. `^0.minor.0` for 0.x versions). If you need a specific feature (e.g. react hooks), specify the version that introduced the feature (i.e. `^16.8.0`).
- Use `yarn upgrade` to upgrade dependencies (instead of updating the version in `package.json`).

Reasoning: When some dependencies don't work together for some reason, we can look into `package.json` to see our "minimal" requirement for the dependency (and therefore know which dependencies we could downgrade).

So in this case, it's not an actual downgrade, since `yarn.lock` still contains the newest version. But it documents that we don't actually need `0.5.2`.